src/xcode/ENA/ENA/Source/Scenes/RiskLegend/Cell/RiskLegendDotBodyCell.swift

```diff
@@ -5,7 +5,47 @@
 import UIKit
 
 class RiskLegendDotBodyCell: UITableViewCell {
-
-	@IBOutlet weak var dotView: UIView!
 	
+	// MARK: - Init
+	
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+	
+	// MARK: - Overrides
+	
+	override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+		super.init(style: style, reuseIdentifier: reuseIdentifier)
+		// dotView
+		dotView = UIView()
+		dotView.backgroundColor = .enaColor(for: .riskHigh)
+		dotView.layer.cornerRadius = 8
+		dotView.layer.masksToBounds = true
+		dotView.translatesAutoresizingMaskIntoConstraints = false
+		contentView.addSubview(dotView)
+		NSLayoutConstraint.activate([
+			dotView.leadingAnchor.constraint(equalTo: contentView.layoutMarginsGuide.leadingAnchor, constant: 8),
+			dotView.trailingAnchor.constraint(lessThanOrEqualTo: contentView.layoutMarginsGuide.trailingAnchor, constant: -8),
+			dotView.centerYAnchor.constraint(equalTo: contentView.layoutMarginsGuide.centerYAnchor),
+			dotView.heightAnchor.constraint(equalToConstant: 16),
+			dotView.widthAnchor.constraint(equalToConstant: 16)
+		])
+		// label
+		label = ENALabel()
+		label.style = .body
+		label.translatesAutoresizingMaskIntoConstraints = false
+		contentView.addSubview(label)
+		NSLayoutConstraint.activate([
+			label.leadingAnchor.constraint(equalTo: dotView.trailingAnchor, constant: 24),
+			label.trailingAnchor.constraint(lessThanOrEqualTo: contentView.layoutMarginsGuide.trailingAnchor, constant: -8),
+			label.centerYAnchor.constraint(equalTo: contentView.layoutMarginsGuide.centerYAnchor),
+			label.topAnchor.constraint(greaterThanOrEqualTo: contentView.layoutMarginsGuide.topAnchor, constant: 8),
+			label.bottomAnchor.constraint(lessThanOrEqualTo: contentView.layoutMarginsGuide.bottomAnchor, constant: -8)
+		])
+	}
+	
+	// MARK: - Internal
+	
+	var dotView: UIView!
```

Comment:
Could be:
```suggestion
	let dotView = UIView()
```

Reply:
Or even:
```suggestion
	let dotView: UIView {
	let view = UIView()
	view.backgroundColor = .enaColor(for: .riskHigh)
	…
	}()
```

This would uncluttered the long init.

Reply:
If I see this correctly now dotView could even be private? 🤔